src/metrics/exemplar.rs

```diff
@@ -31,12 +32,46 @@ pub struct Exemplar<S, V> {
 /// counter_with_exemplar.inc_by(1, Some(vec![("user_id".to_string(), "42".to_string())]));
 /// let _value: (u64, _) = counter_with_exemplar.get();
 /// ```
+/// You can also use exemplars with families. Just wrap the exemplar in a Family.
+/// ```
+/// # use prometheus_client::metrics::exemplar::CounterWithExemplar;
+/// # use prometheus_client::metrics::histogram::exponential_buckets;
+/// # use prometheus_client::metrics::family::Family;
+/// # use prometheus_client_derive_encode::Encode;
+/// #[derive(Clone, Hash, PartialEq, Eq, Encode, Debug, Default)]
+/// pub struct ResultLabel {
+///     pub result: String,
+/// }
+///
+/// #[derive(Clone, Hash, PartialEq, Eq, Encode, Debug, Default)]
+/// pub struct TraceLabel {
+///     pub trace_id: String,
+/// }
+///
+/// let latency: Family<ResultLabel, CounterWithExemplar<TraceLabel>> =
+///     Family::default();
+///
+/// latency
+/// .get_or_create(&ResultLabel {
+///     result: "success".to_owned(),
+/// })
+/// .inc_by(
+///     1,
+///     Some(TraceLabel {
+///         trace_id: "3a2f90c9f80b894f".to_owned(),
+///     }),
+/// );
```

Comment:
Can you fix the indentation here? As far as I know this will propagate into the rendered HTML on the docs.